numpyro/distributions/continuous.py

```diff
@@ -2444,3 +2444,110 @@ def cdf(self, value):
 
     def icdf(self, value):
         return self._ald.icdf(value)
+
+
+class ZeroSumNormal(Distribution):
```

Comment:
Perhaps add a mention that this is adapted from the PyMC implementation?

Reply:
@ricardoV94 do y'all have an official reference on the ZeroSumNormal I can link? Otherwise I figure I could reference the docs? https://www.pymc.io/projects/docs/en/stable/api/distributions/generated/pymc.ZeroSumNormal.html

Reply:
A permalink to the github code may be better

Reply:
I added both for now. Let me know if there's anything else I can do, and I'll be sure to tag you again when its a little closer to its finalized stage

Reply:
Nice @kylejcaron !! Great to see ZSN is spreading 🤩 
You can also link to an episode I did with Adrian, where we go through the origins and utility of ZSN: https://learnbayesstats.com/episode/74-optimizing-nuts-developing-zerosumnormal-distribution-adrian-seyboldt/

Reply:
absolutely, will do! I found that episode very helpful